feedparser/encodings.py

```diff
@@ -32,11 +32,10 @@
 
 try:
     try:
-        import cchardet as chardet
+        import chardet as chardet
     except ImportError:
         import chardet
```

Comment:
It looks like this loses the cchardet import and tries to import chardet twice. Would you double-check this behavior?

Reply:
I didn't actually realise this was a different library as I thought it was a spelling mistake I think! Out of curiosity: why are neither listed as dependencies of this library?

Reply:
It's optional -- developers can install these libraries for additional flexibility but they're not required.